src/lib/userStorage/UserStorageClass.js

```diff
@@ -546,6 +530,7 @@ export class UserStorage {
       try {
         // firstly, awaiting for wallet is ready
         await wallet.ready
+        await this.initUserProfileStorage()
```

Comment:
this needs to be in initregistered
only registered users have jwt with access to realmdb

Reply:
Unfortunately this is not working in init Registered that's why i'm initialize db in there
`  async initUserProfileStorage() {
    try {
      const seed = this.wallet.wallet.eth.accounts.wallet[this.wallet.getAccountForType('gundb')].privateKey.slice(2)

      await this.feedDB.init(seed, this.wallet.getAccountForType('gundb')) //only once user is registered he has access to realmdb via signed jwt
      await this.initFeed()

      this.profileStorage = new UserProfileStorage(this.wallet, this.feedDB)
      await this.profileStorage.init()
    } catch (error) {
      logger.error('initUserProfileStorage failed', error)
    }
  }`